packages/web/intl/locales/en-en.json

```
@@ -1067,52 +1067,52 @@
+  "portfolio.asset.payouts-from-neu.token": "Token",
   "portfolio.asset.payouts-from-neu.total-payout": "Total payout",
+  "portfolio.asset.payouts-from-neu.totals": "Totals",
   "portfolio.asset.payouts-from-neu.your-share": "Your share",
-  "portfolio.section.asset-portfolio.title": "Asset portfolio",
+  "portfolio.section.asset-portfolio.title": "Pending payouts from NEU",
+  "portfolio.section.asset-portfolio.tooltip": "Learn more about NEU tokens at: {link}",
   "portfolio.section.my-assets.download-agreements": "Agreements",
   "portfolio.section.my-assets.modal.header": "Download agreements",
   "portfolio.section.my-assets.send": "Send",
-  "portfolio.section.my-assets.table.header.current-price": "Current price",
-  "portfolio.section.my-assets.table.header.current-value": "Current value",
-  "portfolio.section.my-assets.table.header.quantity": "Quantity",
+  "portfolio.section.my-assets.table.footer.totals": "Totals",
+  "portfolio.section.my-assets.table.header.current-price": "Token price",
+  "portfolio.section.my-assets.table.header.current-value": "Value",
+  "portfolio.section.my-assets.table.header.quantity": "Qty",
   "portfolio.section.my-assets.table.header.token": "Token",
-  "portfolio.section.past-investments.description": "These are assets which have been settled",
-  "portfolio.section.past-investments.table.header.date": "Date",
-  "portfolio.section.past-investments.table.header.eto-status": "Status",
+  "portfolio.section.past-investments.no-investments": "You have no past investments in your portfolio.",
   "portfolio.section.past-investments.table.header.neu-reward": "NEU reward",
-  "portfolio.section.past-investments.table.header.placeholder": "You have currently no past investments in your Portfolio.",
-  "portfolio.section.past-investments.table.header.price-eur": "Purchase price",
-  "portfolio.section.past-investments.table.header.quantity": "Quantity",
+  "portfolio.section.past-investments.table.header.price-eur": "Token price",
+  "portfolio.section.past-investments.table.header.quantity": "QTY",
   "portfolio.section.past-investments.table.header.token": "Token",
-  "portfolio.section.past-investments.table.header.value-eur": "Amount invested",
+  "portfolio.section.past-investments.table.header.value-eur": "Value",
   "portfolio.section.past-investments.title": "Past investments",
+  "portfolio.section.past-investments.tooltip": "These are assets which have been settled",
   "portfolio.section.reserved-assets.claim-tokens": "Claim Tokens",
-  "portfolio.section.reserved-assets.description": "These are assets which you have invested in, but which have not yet been settled.",
-  "portfolio.section.reserved-assets.refund": "Refund",
-  "portfolio.section.reserved-assets.table.header.balance": "Quantity",
-  "portfolio.section.reserved-assets.table.header.eto-status": "ETO Status",
+  "portfolio.section.reserved-assets.ends-in": "{percentageFunded} funded (Ends in {endsIn})",
```

```suggestion
  "portfolio.section.reserved-assets.ends-in": "{percentageFunded} funded (Ends {endsIn})",
```

doubled in
38% funded (Ends in `in` 27 days)
https://github.com/Neufund/platform-frontend/pull/4089/files#diff-21c058ef4fdb5b0d2183a0b83bd43f6dR1282